src/witan/send/adroddiad/pptx/slides.clj

```diff
@@ -172,6 +184,12 @@
     :x margin :y 100
     :bold? true
     :font-size 70.0}
+   {:slide-fn :text-box
+    :text (:sub-title conf)
+    :width slide-minus-margins
+    :x margin :y 225
+    :bold? true
+    :font-size 40.0}
```

Comment:
again, does all of this lock us into having sub titles all of the time?

Reply:
No, but this may be adding whitespace, less clear in pptx